lib/util/package.py

```diff
@@ -0,0 +1,50 @@
+import os
+from pathlib import Path
+
+from .. import versions
+from .subprocess import subprocess_run
+
+
+def download_source(package):
+    """
+    Download src rpm.
+
+    'package' is name of package to download.
+    """
+    cmd = ['yumdownloader', '--source', package]
+    subprocess_run(cmd, check=True)
+    package_glob = package + "*.src.rpm"
+    try:
+        src_path = next(Path('.').glob(package_glob))
+    except StopIteration:
+        return None
+    return str(src_path)
+
+
+def build_source(src_rpm, path=None):
+    """
+    Build package from src rpm.
+
+    'src_rpm' is path to src rpm file.
+    Define 'path' to specify path where to build. Otherwise, build
+    is done to rpmbuild directory in current working directory.
+    """
+    if versions.rhel == 7:
+        yum_builddep = ['yum-builddep']
+    else:
+        yum_builddep = ['dnf', 'builddep']
+
+    cmd = [*yum_builddep, '-y', src_rpm]
+    subprocess_run(cmd, check=True)
+
+    if not path:
+        path = os.getcwd() + "/rpmbuild"
```

Comment:
Given that our current "policy" is to use `pathlib` where possible, this would be more preferable:
```python
path = Path(path) if path else Path.cwd() / 'rpmbuild'
```